src/commands/application/instances/mod.rs

```diff
@@ -32,11 +32,13 @@ pub enum InstancesSubcommand {
     },
     /// Start the interactive session to connect to the remote Elixir instance.
     Connect {
-        /// The instance name to connect to.
-        name: String,
-        /// (optional) The port for livebook instance.
-        #[arg(long, short, default_value_t = 8080)]
-        port: u16,
+        /// (optional) The namespace to list the running instances.
+        #[arg(long, value_enum)]
+        namespace: Option<String>,
```

Comment:
This should be `enum` instead of `String`, it will show the possible values in `--help` as well as show nice error message if user input wrong value
<img width="1449" alt="Screenshot 2023-07-14 at 12 29 45 PM" src="https://github.com/mindvalley/wukong-cli/assets/7545747/82b61a49-fe1a-4098-bbcb-bf9c75317fee">
<img width="1043" alt="Screenshot 2023-07-14 at 12 30 43 PM" src="https://github.com/mindvalley/wukong-cli/assets/7545747/c485a374-da2d-409e-9c2f-632427c636f6">